app/src/main/java/org/gdg/frisbee/android/api/PlusImageResizer.java

```diff
@@ -0,0 +1,41 @@
+package org.gdg.frisbee.android.api;
+
+import org.gdg.frisbee.android.api.model.plus.ImageInfo;
+import org.gdg.frisbee.android.app.App;
+
+import java.io.IOException;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import okhttp3.Interceptor;
+import okhttp3.Request;
+import okhttp3.Response;
+
+public class PlusImageResizer implements Interceptor {
+
+    private static final Pattern mPlusPattern
+        = Pattern.compile("http[s]?:\\/\\/plus\\..*google\\.com.*(\\+[a-zA-Z] +|[0-9]{21}).*");
+
+    @Override
+    public Response intercept(Chain chain) throws IOException {
+        Request request = chain.request();
+
+        Matcher matcher = mPlusPattern.matcher(request.url().toString());
```

Comment:
You removed the ModelCache from here. I think it is really needed. This is called in a ListView to load images from Google+. So cache would be awesome here.